EnglishFilter/sections/content_blocker.txt

```diff
@@ -500,6 +500,8 @@ aioboot.com,dist-tutor.info,investmentwatchblog.com,life.ru,1000ps.de,abw.by,abw
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39571
 razorlink.us#@#.adsbygoogle
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=razorlink.us
+! https://github.com/AdguardTeam/AdguardFilters/issues/39369
+||thelibertydaily.com/*/adtoniq/js/niqon.js
```

Comment:
Not seeing adblock detection.

Reply:
Well, on my iPhone I can't reproduce it too. Rule was deleted.